WebForms/packages.config

```diff
@@ -1,5 +1,9 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="Microsoft.AspNetCore.Html.Abstractions" version="2.2.0" targetFramework="net461" />
-  <package id="System.Text.Encodings.Web" version="4.5.1" targetFramework="net461" />
+  <package id="System.Buffers" version="4.5.1" targetFramework="net461" />
+  <package id="System.Memory" version="4.5.5" targetFramework="net461" />
+  <package id="System.Numerics.Vectors" version="4.5.0" targetFramework="net461" />
+  <package id="System.Runtime.CompilerServices.Unsafe" version="5.0.0" targetFramework="net461" />
+  <package id="System.Text.Encodings.Web" version="5.0.1" targetFramework="net461" />
 </packages>
```

Comment:
All the PR looks ok. Just one question: Did you test having a .NET Core 3.1+ project created and using WebTemplate and then updating to this version to make sure it works?  (or do you need this PR merged to test this)